src/styles/blocks/card.css

```diff
@@ -0,0 +1,90 @@
+
+/* #region card design */
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-image {
```

Comment:
Do we really need to use a `product` part in each name?

Reply:
done